usecases/api/mu_mpc.go

```diff
@@ -0,0 +1,108 @@
+package api
+
+import (
+	"github.com/enbility/eebus-go/api"
+)
+
+// Actor: Monitored Unit
+// UseCase: Monitoring of Power Consumption
+type MuMPCInterface interface {
+	api.UseCaseInterface
+
+	// Scenario 1
+
+	// set the momentary active power consumption or production
+	//
+	// parameters:
+	//   - power: the active power
+	SetPower(power float64) error
+
+	// return the momentary active power consumption or production
+	//
+	// possible errors:
+	//   - ErrDataNotAvailable if no such value is (yet) available
+	//   - and others
+	Power() (float64, error)
+
+	// set the momentary active phase specific power consumption or production per phase
+	//
+	// parameters:
+	//   - phaseA: the active power of phase A
+	//   - phaseB: the active power of phase B
+	//   - phaseC: the active power of phase C
+	SetPowerPerPhase(phaseA, phaseB, phaseC float64) error
+
+	// return the momentary active phase specific power consumption or production per phase
+	//
+	// possible errors:
+	//   - ErrDataNotAvailable if no such values are (yet) available
+	//   - and others
+	PowerPerPhase() ([]float64, error)
```

Comment:
it is obvious that the returned slice contains values for phases A, B, C but I would prefer adding some phase identifiers as well.
With this change, you can provide additional functionality like searching for a specific phase or checking if the value in a particular phase is available if only 1 or 2 values of 3 or more are returned.

Reply:
The current implementation aligns with the API structure of the other use cases. If this should be changed, it should be changed everywhere. See e.g.  the CEM use case implementations e.g. https://github.com/enbility/eebus-go/blob/dev/usecases/api/cem_opev.go

Reply:
Hmm from a user of this lib point of view this could be improved but agreed, this is then not part of this Pull Request.

Thx for clarifying